### DISCUSSION TITLE
Websites Blocked while on Premium

### DISCUSSION THREAD

**Discussion by clarkson22** [vpnhood/VpnHood#422] [Q&A]
I have purchased a hidden server, but all websites that are protected by Cloudflare flag my connection and prevent access. I have tried UDP and non-UDP. Still blocked. Any way to circumvent this?

**Comment by trudyhood** [comment]
Let us check it out

**Comment by trudyhood** [comment]
I can see that everything is working properly on the website that is protected by Cloudflare through our hidden servers. It's possible that your target has been configured to reject IP addresses from cloud servers. Would you be able to share the website with us so that we can check it as well? support@vpnhood.com

**Reply by clarkson22** [reply]
> It's possible that your target has been configured to reject IP addresses from cloud servers

That's probably it. The website I was trying to access was Curseforge.
